src/main/scala/nl/biopet/utils/io/package.scala

```diff
@@ -101,4 +174,17 @@ package object io {
     lines.foreach(writer.println)
     writer.close()
   }
+
+  /**
+    * Calculates the sha256sum of a file that is downloaded from the URL
+    * @param url the URL
+    * @return the hex of the sha256sum.
+    */
+  def getSha256SumFromDownload(url: URL): Option[String] = {
+    try {
+      Some(url.openStream().sha256.hex)
+    } catch {
+      case e: java.io.FileNotFoundException => None
```

Comment:
Not sure you want to catch this, maybe at a message to the exception and throw it again?
You could do things like this:
```scala
case e: FileNotFoundException => 
  throw new FileNotFoundException(s"Could not generate Sha256Sum on ${url}", e)
```
This way you extent the exception.

While writing this it's maybe also good to have a `def getSha256SumFromDownload(url: URL): String` method with this behavior.